packages/framework/src/Component/UploadedFile/UploadedFileRepository.php

```diff
@@ -81,4 +79,28 @@ public function getById(int $uploadedFileId): UploadedFile
 
         return $uploadedFile;
     }
+
+    /**
+     * @param int $uploadedFileId
+     * @param string $uploadedFileSlug
+     * @param string $uploadedFileExtension
+     * @return \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFile
+     */
+    public function getByIdSlugAndExtension(int $uploadedFileId, string $uploadedFileSlug, string $uploadedFileExtension): UploadedFile
+    {
+        $uploadedFile = $this->getUploadedFileRepository()->findOneBy(
+            [
+                'id' => $uploadedFileId,
+                'slug' => $uploadedFileSlug,
+                'extension' => $uploadedFileExtension,
+            ]
+        );
+
+        if ($uploadedFile === null) {
+            $message = sprintf('UploadedFile with ID "%s", slug "%s" and extension "%s" does not exist.', $uploadedFileId, $uploadedFileSlug, $uploadedFileExtension);
```

Comment:
Another nitpick here: Do you mind putting this `sprintf` directly as a exception argument and use multiline formatting? Even now the row is too long.

Reply:
Only one nitpick per one line of code at the PR :-P 

And I think it is ok right now, it is easy to read for me.

Reply:
Do not you mind you need to scroll so far to se what is in?

Reply:
No I do not, but I fixed it for you